packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm

```diff
@@ -287,14 +287,20 @@ - (void)updateProps:(const Props::Shared &)props oldProps:(const Props::Shared &
   }
 
   // `border`
-  if (oldViewProps.borderStyles != newViewProps.borderStyles || oldViewProps.borderRadii != newViewProps.borderRadii ||
+  if (oldViewProps.borderStyles != newViewProps.borderStyles ||
       oldViewProps.borderColors != newViewProps.borderColors) {
     needsInvalidateLayer = YES;
   }
-    
+// 'borderRadii'
+    if (oldViewProps.borderRadii != newViewProps.borderRadii) {
+        needsInvalidateLayer = YES;
+        CGFloat borderRadius = newViewProps.borderRadii.all ?newViewProps.borderRadii.all.value() : 0.0;
```

Comment:
What about the case where we have different border radious on each corner? I think this should also be handled here

Reply:
yes, currently it supports only borderRadius. just want to confirm the proposed solution is correct. I will work on the other cases.